pgscm/templates/admin/macro.html

```diff
@@ -0,0 +1,20 @@
+{% import "bootstrap/wtf.html" as wtf %}
+
+{% macro render_btn(id, size='btn-xs', type='btn-default', modal_target='modal', title="", css_class='plus', data=None) -%}
```

Comment:
because user form have multi select for roles and have password field - cannot pass data into when edit form. I will try to custom to reuse for another form.